tests/forte/data/data_store_test.py

```diff
@@ -248,20 +247,47 @@ def setUp(self) -> None:
             ]
         }
 
+    def test_get_type_info(self):
+        # initialize
+        empty_data_store = DataStore()
+
+        # test add new type to _type_attributes
+        doc_attr_dict = empty_data_store._get_type_info("ft.onto.base_ontology.Document")
+        empty_data_store._get_type_info("ft.onto.base_ontology.Sentence")
+        self.assertEqual(len(empty_data_store._DataStore__elements), 0)
+        self.assertEqual(empty_data_store._type_attributes["ft.onto.base_ontology.Sentence"],
+                        self.data_store._type_attributes["ft.onto.base_ontology.Sentence"])
+        self.assertEqual(empty_data_store._type_attributes["ft.onto.base_ontology.Document"],
+                        self.data_store._type_attributes["ft.onto.base_ontology.Document"])
+        # test the return value
+        self.assertEqual(doc_attr_dict, empty_data_store._type_attributes["ft.onto.base_ontology.Document"]["attributes"])
+        # test add invalid type
+        with self.assertRaisesRegex(
+            ValueError, "Class not found in invalid.Type"
+        ):
+            self.data_store._get_type_info("invalid.Type")
+        self.assertTrue("invalid.Type" not in self.data_store._type_attributes)
```

Comment:
We will need more tests for dealing with types in python class and ontology files (such as missing type in both, type appear in both, and other corner cases)

Reply:
The parse ontology file function is not merged yet, so I guess we cannot test that part for now

Reply:
yeah, just try to make sure we remember to test these.

Reply:
need to create issues to track these otherwise they may be forgotten.